corehq/messaging/smsbackends/twilio/models.py

```diff
@@ -1,11 +1,17 @@
-from corehq.apps.sms.models import SQLSMSBackend, PhoneLoadBalancingMixin, SMS
-from corehq.messaging.smsbackends.twilio.forms import TwilioBackendForm
 from twilio.base.exceptions import TwilioRestException
 from twilio.rest import Client
-from django.conf import settings
 
-#https://www.twilio.com/docs/api/errors/reference
+from dimagi.utils.logging import notify_exception
+
+from corehq import toggles
+from corehq.apps.sms.models import SMS, PhoneLoadBalancingMixin, SQLSMSBackend
+from corehq.messaging.smsbackends.twilio.forms import TwilioBackendForm
+
+# https://www.twilio.com/docs/api/errors/reference
 ERROR_INVALID_TO_PHONE_NUMBER = 21211
+ERROR_WHATSAPP_LIMITATION = 63032
```

Comment:
`WHATSAPP_LIMIT_ERROR_CODE` would be better?
And if we can add a comment saying what the error is? No. of messages sent or specifically what this is?

Reply:
It isn't a limit, it's a catch-all error for not being able to send a whatsapp.

Reply:
cool, a comment on that would be helpful in the code.
also, if the naming pattern i suggested felt better, `SOME_THING_ERROR_CODE`